### PR TITLE
Colorize logger function name, update logs.

### PR DESCRIPTION
**Pull request by patuwwy**
image

### CODE REVIEW

File: packages/adapters/src/instance-adapter.ts [hunk 1]
```diff
@@ -270,14 +268,12 @@ IComponent {
...
 
-        this.logger.debug(`Container is running (${containerId})`);
+        this.logger.log(`Container is running (${containerId}).`);
 
         try {
             const { statusCode } = await this.dockerHelper.wait(containerId);
 
-            this.logger.debug("Container exited");
-
-            await defer(10000);
```
Comment (patuwwy) [inline packages/adapters/src/instance-adapter.ts:280]: This should be removed in separate PR

**Review by alicja-gruzdz — APPROVED** [review]
:paintbrush: :art:

**Review by a-tylenda — APPROVED** [review]

File: packages/adapters/src/instance-adapter.ts (hunk 1 above)
Comment (alicja-gruzdz) [reply packages/adapters/src/instance-adapter.ts:280]:
```suggestion
            await defer(10000);
```
